R/zchunk_LA154.transportation_UCD.R

```diff
@@ -40,116 +44,376 @@ module_energy_LA154.transportation_UCD_DISABLED <- function(command, ...) {
 
     # Load required inputs
     iso_GCAM_regID <- get_data(all_data, "common/iso_GCAM_regID")
-    calibrated_techs_trn_agg <- get_data(all_data, "energy/calibrated_techs_trn_agg")
+    calibrated_techs_trn_agg <- get_data(all_data, "energy/mappings/calibrated_techs_trn_agg")
     enduse_fuel_aggregation <- get_data(all_data, "energy/enduse_fuel_aggregation")
-    UCD_ctry <- get_data(all_data, "energy/UCD_ctry")
-    UCD_techs <- get_data(all_data, "energy/UCD_techs")
-
-    # Russell - it looks like skeleton script got messed up on next line
-    # Probably a loop in original file that loads stuff dynamically
-    UCD_trn_data_ <- get_data(all_data, "energy/paste0( UCD_trn_data_")
-
-    L101.in_EJ_ctry_trn_Fi_Yh <- get_data(all_data, "L101.in_EJ_ctry_trn_Fi_Yh")
-    L1011.in_EJ_ctry_intlship_TOT_Yh <- get_data(all_data, "L1011.in_EJ_ctry_intlship_TOT_Yh")
+    UCD_ctry <- get_data(all_data, "energy/mappings/UCD_ctry")
+    UCD_techs <- get_data(all_data, "energy/mappings/UCD_techs")
+    UCD_trn_data <- get_data(all_data, paste0("energy/UCD_trn_data_", energy.TRN_SSP)) %>%
```

Comment:
This is generating a `readr` parsing warning on my system. Need to fix before merging. We can do this either by changing the file(s), or by making a dedicated data chunk to handle this file. Let's talk.